shared/elements/gaia_switch/style.css

```diff
@@ -47,7 +47,7 @@ gaia-switch {
   content: '';
   position: absolute;
   right: 0;
-  top: 50%;
+  top: 2.5rem;
```

Comment:
haven't try in real device yet, but does the `title only` style panel's text position be effect by this change?

(The label position for `gaia-switch > label` & `gaia-switch > label + details` are not the same)